wallet/src/de/schildbach/wallet/ui/SecurityActivity.kt

```diff
@@ -1,244 +0,0 @@
-/*
- * Copyright 2019 Dash Core Group
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package de.schildbach.wallet.ui
-
-import android.app.Activity
-import android.content.Intent
-import android.os.Bundle
-import android.view.View
-import android.view.View.GONE
-import android.view.View.VISIBLE
-import android.widget.CompoundButton
-import androidx.activity.viewModels
-import androidx.appcompat.widget.SwitchCompat
-import androidx.lifecycle.ViewModelProvider
-import androidx.lifecycle.lifecycleScope
-import dagger.hilt.android.AndroidEntryPoint
-import de.schildbach.wallet.ui.backup.BackupWalletDialogFragment
-import de.schildbach.wallet.util.FingerprintHelper
-import de.schildbach.wallet_test.R
-import kotlinx.android.synthetic.main.activity_security.*
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.launch
-import org.bitcoinj.core.Coin
-import org.bitcoinj.wallet.DeterministicSeed
-import org.dash.wallet.common.BuildConfig
-import org.dash.wallet.common.services.analytics.AnalyticsConstants
-import org.dash.wallet.common.ui.dialogs.AdaptiveDialog
-import org.dash.wallet.common.ui.dialogs.ExtraActionDialog
-
-@ExperimentalCoroutinesApi
-@AndroidEntryPoint
-class SecurityActivity : BaseMenuActivity() {
```

Comment:
Moved to ui/more

Reply:
It puzzles me how sometimes git will say a file is moved and other times deleted and created again.  This one is deleted / created in the other folder, while other files in this PR were moved.

Reply:
I think it might have something to do with the number of changes in the moved class. I had replaced synthetics and improved mvvm flows in there so it ended up being a lot.